src/System.IO.Pipelines/tests/PipeReaderWriterFacts.cs

```diff
@@ -209,33 +200,6 @@ public void Dispose()
             Assert.Equal("Reading is not allowed after reader was completed.", exception.Message);
         }
 
-        [Fact]
-        public async Task EmptyBufferStartCrossingSegmentBoundaryIsTreatedLikeAndEnd()
```

Comment:
This test is removed because we are not able to get pipe into a state required without a feature that is being removed.